proposals/compatibility-mode.md

```diff
@@ -197,6 +197,18 @@ generate a validation error.
 other APIs only support the first vertex so only `@interpolation(flat, either)` is supported in
 compatibility mode.
 
+## 18. Introduce new `maxVertexStageStorageBuffers` and `maxVertexStageStorageTextures` limits.
```

Comment:
bikeshedding:
I agree `maxStorageBuffersPerVertexStage` wouldn't make sense since there is only one vertex stage, but as I've been talking about this I find it a little confusing that the word order differs from the old limits. Maybe `maxStorageBuffersInVertexStage`?

Also as I've been typing these comments I keep mistyping as `VertexBuffer` because the words are in that order. That's not really important but just noting.

Reply:
`maxStorageBuffersInVertexStage` sounds better to me as well.